packages/project-disaster-trail/src/state/tasks.js

```diff
@@ -35,10 +44,47 @@ export const addTask = taskChoice => dispatch => {
 export const increaseActiveTask = () => dispatch => {
   dispatch({ type: actionTypes.INCREASE_ACTIVE_TASK });
 };
+export const startTask = task => dispatch => {
+  dispatch({ type: actionTypes.START_TASK, task });
+};
 export const completeTask = completedTask => dispatch => {
   dispatch({ type: actionTypes.COMPLETE_TASK, completedTask });
 };
 
+// summon `startTick` when you want to start a new timer,
+// it uses requestAnimationFrame to summon the `loop` function
+// The loop function evaluates if the timer is expired (outOfTime)
+// or if the timer should 'tick' which counts down.
+// export const startTick = totalTime => (dispatch, getState) => {
+//   let now = new Date();
+//   dispatch({ type: actionTypes.SET_START_TIME, time: now });
+
+//   const loop = () => {
+//     timerId = window.requestAnimationFrame(loop);
+
+//     now = new Date();
+//     const {
+//       tasks: { activeTime, startTime }
+//     } = getState();
+
+//     const outOfTime = activeTime - startTime > totalTime;
+//     if (!outOfTime) {
+//       dispatch({ type: actionTypes.TICK, time: now, totalTime });
+//     } else {
+//       dispatch({ type: actionTypes.OUT_OF_TIME });
+//       window.cancelAnimationFrame(timerId);
+//     }
+//   };
+
+//   window.cancelAnimationFrame(timerId);
+//   timerId = window.requestAnimationFrame(loop);
+// };
+
+// // use when unloading a component that summoned a timer
+// export const stopTick = () => () => {
+//   window.cancelAnimationFrame(timerId);
+// };
+
```

Comment:
It looks like this is something you tried and decided not to use. Can we remove it?